app/helpers/fhir_helper.rb

```diff
@@ -0,0 +1,331 @@
+                                  : nil,
+        patient.secondary_telephone ? FHIR::ContactPoint.new(system: 'phone',
+                                                             value: patient.secondary_telephone,
+                                                             rank: 2,
+                                                             extension: [to_string_extension(patient.secondary_telephone_type, 'phone-type')])
+                                    : nil,
+        patient.email ? FHIR::ContactPoint.new(system: 'email', value: patient.email, rank: 1) : nil
+      ].reject(&:nil?),
+      birthDate: patient.date_of_birth&.strftime('%F'),
+      address: [
+        FHIR::Address.new(
+          line: [patient.address_line_1, patient.address_line_2].reject(&:blank?),
+          city: patient.address_city,
+          district: patient.address_county,
+          state: patient.address_state,
+          postalCode: patient.address_zip
+        )
+      ],
+      communication: [
+        language_coding(patient.primary_language) ? FHIR::Patient::Communication.new(
+          language: FHIR::CodeableConcept.new(coding: [language_coding(patient.primary_language)]),
+          preferred: patient.interpretation_required
+        ) : nil
+      ].reject(&:nil?),
+      extension: [
+        to_us_core_race(patient.white, patient.black_or_african_american, patient.american_indian_or_alaska_native,
+                        patient.asian, patient.native_hawaiian_or_other_pacific_islander),
+        to_us_core_ethnicity(patient.ethnicity),
+        to_us_core_birthsex(patient.sex),
+        to_string_extension(patient.preferred_contact_method, 'preferred-contact-method'),
+        to_string_extension(patient.preferred_contact_time, 'preferred-contact-time'),
+        to_date_extension(patient.symptom_onset, 'symptom-onset-date'),
+        to_date_extension(patient.last_date_of_exposure, 'last-date-of-exposure'),
+        to_bool_extension(patient.isolation, 'isolation'),
+        to_string_extension(patient.jurisdiction.jurisdiction_path_string, 'full-assigned-jurisdiction-path'),
+        to_string_extension(patient.monitoring_plan, 'monitoring-plan'),
+        to_positive_integer_extension(patient.assigned_user, 'assigned-user'),
+        to_date_extension(patient.additional_planned_travel_start_date, 'additional-planned-travel-start-date'),
+        to_string_extension(patient.port_of_origin, 'port-of-origin'),
+        to_date_extension(patient.date_of_departure, 'date-of-departure'),
+        to_string_extension(patient.flight_or_vessel_number, 'flight-or-vessel-number'),
+        to_string_extension(patient.flight_or_vessel_carrier, 'flight-or-vessel-carrier'),
+        to_date_extension(patient.date_of_arrival, 'date-of-arrival'),
+        to_string_extension(patient.exposure_notes, 'exposure-notes'),
```

Comment:
Based on the discussion in sprint review today - it sounds like we want to change this to just "notes" for now since it shows up in both workflows.

Reply:
Sure I can do that. Import will still be "Exposure Notes" though (just mentioning since unifying the FHIR extension IDs with Import/UI was something discussed earlier in this PR).

Reply:
Mm, you know what maybe that change can happen here when it happens to the import so they always stay consistent. Let's just keep an eye on that and whenever we get a PR for that in, let's make sure it includes this.